tests/.settings/org.eclipse.jdt.ui.prefs

```diff
@@ -40,6 +40,8 @@ sp_cleanup.qualify_static_member_accesses_through_subtypes_with_declaring_class=
 sp_cleanup.qualify_static_member_accesses_with_declaring_class=true
 sp_cleanup.qualify_static_method_accesses_with_declaring_class=false
 sp_cleanup.remove_private_constructors=true
+sp_cleanup.remove_redundant_modifiers=false
+sp_cleanup.remove_redundant_semicolons=false
```

Comment:
These lines looks unrelated to the commit message (and are only added in this file).

Reply:
Good catch. Fixed.

Reply:
Fixed (And what is the button "Resolve conversation" doing, which I see below this input box? I never saw that before.)

Reply:
Resolved conversations are hidden by default.